public/wp-content/themes/newsup-child/footer.php

```diff
@@ -21,7 +22,14 @@
                 </div>
             </div>
             <?php }
-            $newsup_you_missed_loop = new WP_Query(array( 'post_type' => 'post', 'posts_per_page' => 4, 'order' => 'DESC',  'ignore_sticky_posts' => true));
+            $you_missed_args = array(
+	            'post_type' => 'anime',
+                'posts_per_page' => 4,
+                'order' => 'DESC',
+                'orderby' => 'rand',
```

Comment:
Put this is order of newest to oldest, make it how many ever fit on a full screen medium viewport,

Reply:
and change it to "You May have missed" instead of "You missed"